packages/config/config/devices/0x0371/zwa038.json

```
@@ -2,7 +2,7 @@
 	"manufacturer": "Aeotec Ltd.",
 	"manufacturerId": "0x0371",
 	"label": "ZWA038",
-	"description": "illumino Wall Switch",
+	"description": "Illumino Wall Switch",
```

AFAIK they are called "illumino" (lowercase)
https://aeotec.com/z-wave-wall-switch/index.html

```suggestion
	"description": "illumino Wall Switch",
```